app/src/main/java/com/example/nsc_events/screen/Login.kt

```diff
@@ -43,178 +52,202 @@ import androidx.compose.ui.text.style.TextDecoration
 import androidx.compose.ui.unit.dp
 import androidx.compose.ui.unit.sp
 import androidx.navigation.NavHostController
+import androidx.navigation.compose.rememberNavController
 import com.example.nsc_events.R
 import com.example.nsc_events.Routes
 
+@OptIn(ExperimentalMaterial3Api::class)
 @Composable
 fun LoginPage(navController: NavHostController) {
     var email by remember { mutableStateOf("") }
     var password by remember { mutableStateOf("") }
     var isEmailValid by remember { mutableStateOf(true) }
     var isPasswordValid by remember { mutableStateOf(true) }
 
-    Column(
+    val navController = rememberNavController()
```

Comment:
for some reason the application keeps closing every time I press the back button from the login page. I think it might be this extra navController variable. 
@tinpham5614 are you able to navigate back to the home page from the login page?

Reply:
You are right! I removed that line of code and it works again. Thank you for pointing this out, @Robel-003.
@kkennedy-gh Can you please remove line 67 that causes a bug?

Reply:
Yes, thank you Robel for pointing the error out.